src/src/rx_main.cpp

```diff
@@ -448,15 +448,14 @@ bool ICACHE_RAM_ATTR HandleSendTelemetryResponse()
 
     OtaGeneratePacketCrc(&otaPkt);
 
-    SX12XX_Radio_Number_t transmittingRadio = geminiMode ? SX12XX_Radio_All : SX12XX_Radio_Default;
-    SX12XX_Radio_Number_t clearChannelsMask = SX12XX_Radio_All;
+    SX12XX_Radio_Number_t transmittingRadio = geminiMode ? SX12XX_Radio_All : Radio.GetLastSuccessfulPacketRadio();
+
 #if defined(Regulatory_Domain_EU_CE_2400)
-    clearChannelsMask = ChannelIsClear(transmittingRadio);
-    if (clearChannelsMask)
+    transmittingRadio &= ChannelIsClear(transmittingRadio);   // weed out the radio(s) if channel in use
+    if (transmittingRadio)
```

Comment:
Remove this `if` so that the `TXnbISR` is still called like in tx_main.

Reply:
This was actually done intentionally to be as close to the pre-PR code as possible. I have taken this out but suggest testing for side effects (connection issues, telemetry issues, etc)